lib/grape/validations/validators/length_validator.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+module Grape
+  module Validations
+    module Validators
+      class LengthValidator < Base
+        def initialize(attrs, options, required, scope, **opts)
+          @min = options[:min]
+          @max = options[:max]
+
+          super
+
+          raise ArgumentError, "min #{@min} cannot be greater than max #{@max}" if !@min.nil? && !@max.nil? && @min > @max
```

Comment:
Should we handle negative values here too?

Reply:
done restricted allowed values to non negative integer numbers.